### DISCUSSION TITLE
Kernel optimizations from CachyOS

### DISCUSSION THREAD

**Discussion by OpenSourceAnarchist** [AerynOS/discussions#13] [Ideas]
Long-time user of Solus years ago, looking forward to using SerpentOS. Currently using CachyOS which just put out a blog post called [Optimizing the Kernel with AutoFDO on CachyOS](https://cachyos.org/blog/2411-kernel-autofdo/). I just figured I'd link it here in case it's different from the Clear Linux and your own optimizations! :)

**Comment by ikeycode** [comment]
Definitely worth investigating. The autofdo stuff is very clear linux fwiw ^^
